### PR TITLE
Proposed single publisher experimental design.

### PR DESCRIPTION
**Pull request by matthewclegg**

### CODE REVIEW

**Review by jiayu-google — APPROVED** [review]
LGTM!

File: src/driver/experimental_trial.py [hunk 1]
```diff
@@ -125,9 +125,9 @@ def evaluate(self, rng: np.random.Generator) -> pd.DataFrame:
             halo, self._trial_descriptor.system_params, privacy_budget
         )
 
-        test_points = self._trial_descriptor.experiment_params.generate_test_points(
+        test_points = list(self._trial_descriptor.experiment_params.generate_test_points(
```
Comment (riemanli) [inline src/driver/experimental_trial.py:128]: https://github.com/world-federation-of-advertisers/planning-evaluation-framework/blob/26ef53422c3ab83b6336cd656643468770536998/src/driver/experiment_parameters.py#L67

If ```generate_test_points``` doesn't return List, the above typing description should be modified.

File: src/simulator/single_publisher_strategy.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+from wfa_planning_evaluation_framework.simulator.halo_simulator import HaloSimulator
+from wfa_planning_evaluation_framework.simulator.modeling_strategy import (
+    ModelingStrategy,
+)
+from wfa_planning_evaluation_framework.simulator.privacy_tracker import PrivacyBudget
+from wfa_planning_evaluation_framework.simulator.system_parameters import (
+    SystemParameters,
+)
+from wfa_planning_evaluation_framework.simulator.privacy_tracker import (
+    PrivacyBudget,
```
Comment (riemanli) [inline src/simulator/single_publisher_strategy.py:30]: Is there any particular reason to import `PrivacyBudget` twice?

File: src/driver/experimental_trial.py (hunk 1 above)
Comment (matthewclegg) [reply src/driver/experimental_trial.py:128]: Thanks for pointing this out.  I changed the signature to return Iterable[ReachPoint].

File: src/simulator/single_publisher_strategy.py (hunk 2 above)
Comment (matthewclegg) [reply src/simulator/single_publisher_strategy.py]: No.  Thanks for catching that.

**Review by riemanli — APPROVED** [review]

**Review by RonPinelliMRC — APPROVED** [review]
MRC approving based on cursory review after confirming non-independent code review on an exception basis. Experimental code will require independent review prior to publication of or reliance on results generated.

**Pull request merged**